src/com/xilinx/rapidwright/util/VivadoTools.java

```diff
@@ -473,4 +473,42 @@ public static float getWorstSetupSlack(Path dcp, Path workdir, boolean encrypted
         }
         return Float.NaN;
     }
+
+    /**
+     * Open a netlist in Vivado and write a new DCP.
+     *
+     * @param dcp        Path to DCP to open in Vivado.
+     * @param outputName Name of the output design.
+     * @param workdir    Directory to work within.
+     * @param encrypted  Indicates whether DCP contains encrypted EDIF cells.
+     * @return Path to output dcp after round-trip.
+     */
+    public static Path roundTripDCPInVivado(Path dcp, String outputName, Path workdir, boolean encrypted) {
```

Comment:
What do you think about refactoring the name to something like `roundTripDCPThruVivado()`?